otlp/go.mod

```diff
@@ -0,0 +1,10 @@
+module go.opentelemetry.io/proto/otlp
```

Comment:
Do we use vanity url for this repo? I don't see `go.opentelemetry.io/proto` enabled.

Reply:
[Not yet](https://github.com/open-telemetry/opentelemetry-go-vanityurls/pull/11).  I was waiting until there is usable logic in this repository to enable that URL.